node-red-project/settings.js

```diff
@@ -21,541 +21,540 @@
+   * Available languages include: en-US (default), ja, de, zh-CN, zh-TW, ru, ko
+   * Some languages are more complete than others.
+   */
+  // lang: "de",
+
+  /** Configure diagnostics options
+   * - enabled:  When `enabled` is `true` (or unset), diagnostics data will
+   *   be available at http://localhost:1880/diagnostics
+   * - ui: When `ui` is `true` (or unset), the action `show-system-info` will
+   *   be available to logged in users of node-red editor
+   */
+  diagnostics: {
+    /** enable or disable diagnostics endpoint. Must be set to `false` to disable */
+    enabled: true,
+    /** enable or disable diagnostics display in the node-red editor. Must be set to `false` to disable */
+    ui: true,
+  },
+  /** Configure runtimeState options
+   * - enabled:  When `enabled` is `true` flows runtime can be Started/Stopped
+   *   by POSTing to available at http://localhost:1880/flows/state
+   * - ui: When `ui` is `true`, the action `core:start-flows` and
+   *   `core:stop-flows` will be available to logged in users of node-red editor
+   *   Also, the deploy menu (when set to default) will show a stop or start button
+   */
+  runtimeState: {
+    /** enable or disable flows/state endpoint. Must be set to `false` to disable */
+    enabled: false,
+    /** show or hide runtime stop/start options in the node-red editor. Must be set to `false` to hide */
+    ui: false,
+  },
+  /** Configure the logging output */
+  logging: {
+    /** Only console logging is currently supported */
+    console: {
+      /** Level of logging to be recorded. Options are:
+       * fatal - only those errors which make the application unusable should be recorded
+       * error - record errors which are deemed fatal for a particular request + fatal errors
+       * warn - record problems which are non fatal + errors + fatal errors
+       * info - record information about the general running of the application + warn + error + fatal errors
+       * debug - record information which is more verbose than info + info + warn + error + fatal errors
+       * trace - record very detailed logging + debug + info + warn + error + fatal errors
+       * off - turn off all logging (doesn't affect metrics or audit)
+       */
+      level: "info",
+      /** Whether or not to include metric events in the log output */
+      metrics: false,
+      /** Whether or not to include audit events in the log output */
+      audit: false,
+    },
+  },
+
+  /** Context Storage
+   * The following property can be used to enable context storage. The configuration
+   * provided here will enable file-based context that flushes to disk every 30 seconds.
+   * Refer to the documentation for further options: https://nodered.org/docs/api/context/
+   */
+  contextStorage: {
+    //    default: {
+    //        module:"localfilesystem"
+    //    }
+    // used for storing data entered on the Configuration UI tab
+    config: {
```

Comment:
The purpose is for this to survive device restarts and container restarts, right?  If so, you will need to Configure this module to specify a container folder that is specified and mounted as a persistent Docker volume.
* https://nodered.org/docs/api/context/store/localfilesystem
* https://docs.docker.com/storage/volumes/